### PR TITLE
Alias volkswagen and Chevrolet in the `/cars` endpoint

### PR DESCRIPTION
**Pull request by jsuna**
Story details: https://app.clubhouse.io/volta/story/2547

I have setup [developer-tools](https://github.com/Volta-Charging/developer-tools/pull/4) to test this locally. NOTE: you'll also need to [`cars-import`](https://github.com/Volta-Charging/cars-import) before local testing..

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
 
 import db from './database'
+import { MAKE_ALIASES } from './config'
+
+function addAliases(response) {
+  let newResponse = []
+  response.map((car) => {
+    const make = car.make.toLowerCase()
+    if (MAKE_ALIASES[make]) {
+      for (let aliasIndx in MAKE_ALIASES[make]) {
```
Comment (cazzer) [inline src/index.js]: I think there is a missing `e` here.

File: src/index.js [hunk 2]
```diff
@@ -1,6 +1,25 @@
 import _get from 'lodash.get'
 
 import db from './database'
+import { MAKE_ALIASES } from './config'
+
+function addAliases(response) {
+  let newResponse = []
+  response.map((car) => {
```
Comment (cazzer) [inline src/index.js]: It might make more sense to use `.forEach` or a `for` loop here since we are not modifying the array.

**Comment by cazzer** [comment]
This looks good to go, once we add code shifting it will be ready to merge.

File: src/index.js [hunk 3]
```diff
@@ -1,6 +1,26 @@
 import _get from 'lodash.get'
 
 import db from './database'
+import { MAKE_ALIASES } from './config'
+
+function addAliases(response) {
+  let newResponse = []
+  
+  response.forEach((car) => {
```
Comment (ste00martin) [inline src/index.js]: only thought was the speed of map being faster than forEach: https://codeburst.io/javascript-map-vs-foreach-f38111822c0f

File: src/index.js [hunk 4]
```diff
@@ -1,6 +1,26 @@
...
+    const make = car.make.toLowerCase()
+    if (MAKE_ALIASES[make]) {
+      for (let aliasIndex in MAKE_ALIASES[make]) {
```
Comment (ste00martin) [inline src/index.js]: could be:
```
newResponse = newResponse.concat(MakeAliases[make].map(newMake => {
let clonedCar = Object.assign({}, car)
clonedCar.make = newMake
return clonedCar
}))
```
fewer concats

**Pull request merged**